src/models/github-server-app.ts

```diff
@@ -109,6 +109,25 @@ export class GitHubServerApp extends EncryptedModel {
 		});
 	}
 
+	/**
+	 * Get one GitHubServerApp for uuid with installationId
+	 *
+	 * @param uuid
+	 * @param installationId
+	 * @returns {GitHubServerApp[]}
```

Comment:
I'd prefer we don't do JSDocs when we're using TS.  TS already has types and definition and they'll conflict with each other in the IDE